app/models/real_sitemap.rb

```diff
@@ -0,0 +1,13 @@
+class RealSitemap < ActiveRecord::Base
```

Comment:
We still want to use `Sitemap` as the name of the class. You need to rename the migration file & migration class (`CreateSitemapsTable`?), not the ActiveRecord class.